app.ts

```diff
@@ -0,0 +1,357 @@
+import express,{Request,Response} from 'express';
+
+
+
+//Import Required Data
+import quiz_json from './quiz.json'
+import data from './response.json'
+import format_json from './format.json'
+import section_json from './section.json'
+
+
+// import {objectOfRsponse} from './interfaces/response'
+
+let response_json = data;
+
+
+const app = express();
+app.use(express.json());
+
+const port:number = 3000;
+
+app.post('/task', solution);
+
+let userSelectedTopicId:number[] = []
+let userSelectedFormatId:number[] = []
+let  userSelectedSourceIds:number[] = []
+
+
+function solution(req:Request, res:Response) {
```

Comment:
you can add all condition in one master function so it will be easy you don't need need to make response loop for every section